search/index.py

```diff
@@ -0,0 +1,17 @@
+import logging
+
+from elasticsearch import Elasticsearch
+from elasticsearch.exceptions import ConnectionError
+
+from search.constants import ELASTICSEARCH_HOSTS
+
+LOGGER = logging.getLogger(__name__)
+
+
+def index_document(index: str, doc_type: str, id: str, body: dict) -> None:
+    try:
+        search = Elasticsearch(hosts=ELASTICSEARCH_HOSTS)
+        search.index(index=index, doc_type=doc_type, id=id, body=body)
+        LOGGER.debug('Document %s saved at /%s/%s/%s', id, index, doc_type, id)
+    except ConnectionError:
+        LOGGER.exception('Unable to connect to %s', ELASTICSEARCH_HOSTS)
```

Comment:
Should this re-raise? Are other exceptions possible too?

Reply:
This indexes a single document. I decided to catch this exception so that other documents could be indexed but I could just let the app crash.

Logger.exception logs the message using logging level ERROR and supplies a stack trace. Devs should then be notified of the error via a tool like Sentry or New Relic if/when implemented.

Reply:
Yes, I mean are there other possible exceptions (eg it connects but won't index an item). If so, it'd probably worth catching them too.